package.json

```diff
@@ -8,5 +8,8 @@
   },
   "keywords": [],
   "author": "Aayush",
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
```

Comment:
Consider adding a start script in the scripts section of your package.json. This will make it easier to start your server. [medium]